feel-engine/src/test/scala/org/camunda/feel/ExternalFunctionsTest.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.feel
+
+import org.camunda.feel.FeelEngine.{Configuration, Failure}
+import org.camunda.feel.syntaxtree.ParsedExpression
+import org.scalatest.{FlatSpec, Matchers}
+
+class ExternalFunctionsTest extends FlatSpec with Matchers {
```

Comment:
Not sure about the test scope and the name of this test class. If this test is only to test the switch, then the content is Ok, but the name promises maybe a little too much.

If this test is for "everything related to external functions", I would also expect test cases and/or assertions where the result of an external function parsing and evaluation is verified.

Also I would expect a small unit test for the ExpressionValidator

Reply:
Do you have a suggestion for a name? This class should only test that external functions can be enabled and disabled.

Reply:
A unit test for the `ExpressionValidator` itself would be a bit clunky to write because it works on the parsed expression. I think it is more stable to use the FEEL syntax directly instead of relying on the syntax tree. 
However, I could cover more cases. I just picked the most obvious case with the context. I think all other cases would also use the context as part of the expression.

Reply:
Something like `ExternalFunctionsFlag/Switch/SettingTest`